### PR TITLE
Add changes for studio's routing

### PR DESCRIPTION
**Pull request by AleksandroBoko**
Fixes for studio's routing. The routes which are used for getting studios with games and ratings were renamed.

### CODE REVIEW

File: GameStore/Controllers/StudioController.cs [hunk 1]
```diff
@@ -17,10 +17,10 @@ public StudioController(IStudioService studioService)
 
         public IEnumerable<StudioModel> GetStudios() => studioService.GetAll();
 
-        [Route("games")]
-        public IEnumerable<StudioInfoTransferModel> GetGames() => studioService.GetStudioInfoTransferAll();
+        [Route("withgames")]
```
Comment (SafeNuts) [inline GameStore/Controllers/StudioController.cs]: Let's make it `with-games` :)

File: GameStore/Controllers/StudioController.cs [hunk 2]
```diff
@@ -17,10 +17,10 @@ public StudioController(IStudioService studioService)
...
+        public IEnumerable<StudioInfoTransferModel> GetStudiosWithGames() => studioService.GetStudioInfoTransferAll();
 
-        [Route("ratings")]
-        public IEnumerable<StudioRateInfo> GetRatings() => studioService.GetAllStudiosRatings();
+        [Route("withratings")]
```
Comment (SafeNuts) [inline GameStore/Controllers/StudioController.cs]: Let's make it with-ratings:)

File: GameStore/Controllers/StudioController.cs (hunk 1 above)
Comment (AleksandroBoko) [reply GameStore/Controllers/StudioController.cs]: Fixed

File: GameStore/Controllers/StudioController.cs (hunk 2 above)
Comment (AleksandroBoko) [reply GameStore/Controllers/StudioController.cs]: Fixed

**Pull request merged**